.flake8/azul_flake8.py

```diff
@@ -30,6 +29,14 @@
     config,
 )
 
+# `scripts/` is not in PYTHONPATH because scripts should not be imported from
```

Comment:
What's missing from this comment is mention of the fact that this emulates Python behavior so that `python dir/scipt.py` dynamically adds `dir` to `sys.path`.

See `<script>` section in https://docs.python.org/3/using/cmdline.html